tests/testthat/_snaps/windows/report.aov.md

```diff
@@ -10,7 +10,6 @@
       
         - The main effect of Species is statistically significant and large (F(2, 147) = 49.16, p < .001; Eta2 = 0.40, 95% CI [0.30, 1.00])
       
-      Effect sizes were labelled following Field's (2013) recommendations.
```

Comment:
@mattansb Was there some change in `{effectsize}` that is removing these recommend statements?

I went through the NEWS for the patch release, but don't see anything that might explain this behavior.

Reply:
I made a small internal change - let me take a look and see if that's what this is.

Reply:
Thanks!